Assets/HoloToolkit/Utilities/Plugins/README.md

```diff
@@ -0,0 +1 @@
+This folder is for the Externals.zip that should alway be up to date.
```

Comment:
But it does mean there is another copy of this uncompilable code in the repo now right? I guess that's what I'm not sure if the best option. So when someone updates the External folder they also now need to update the zip file.